docs/deploying/fly_io.md

```diff
@@ -145,4 +149,10 @@ fly deploy
 Note that it may take a moment for the UDP traffic to be forwarded to the application.
 This means for example, that WebRTC may not be working yet.
 
-This is the tricky part of the deployment, which we weren't able entirely figure out. Sometimes the UDP just works, others it takes ages for it to start flowing.
\ No newline at end of file
+This is the tricky part of the deployment, which we weren't able entirely figure out. Sometimes the UDP just works, others it takes ages for it to start flowing.
```

Comment:
Put each sentence on a new line. This way it's easier to track changes using git.

```suggestion
This is the tricky part of the deployment, which we weren't able to entirely figure out. 
Sometimes the UDP just works, others it takes ages for it to start flowing.
```